src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
 // write code here
+const ulItem = document.querySelector('ul');
+const liItems = document.querySelectorAll('li');
+
+function sortList(list, listItems) {
+  const itemsSorted = [...listItems].sort((a, b) => (
+    +((b.dataset.salary)
+      .slice(1)
+      .split(',')
+      .join(''))
+    - +((a.dataset.salary)
+      .slice(1)
+      .split(',')
+      .join(''))
```

Comment:
Will be more readable if you'll store result of this operations for both salaries to variables and return result of subtraction

Reply:
Also would be nice to move this converting logic to helper function